### ISSUE TITLE
[Bug]: Discord Invalid Invite

### ISSUE THREAD

**Issue by sudo888samewick** [infiniflow/ragflow#227]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Branch name

main

### Commit ID

newest

### Actual behavior

The Discord link failed.
![image](https://github.com/infiniflow/ragflow/assets/150644414/69245db7-3441-4521-85e0-6670938235be)

### Steps to reproduce

```Markdown
Just click here
https://github.com/infiniflow/ragflow?tab=readme-ov-file#-community
The Discord link not work.
```

**Comment by yingfeng** [comment]
We're sorry, the new invitation is updated.

**Comment by JinHai-CN** [comment]
Fixed by #230
